pkgs/development/ocaml-modules/fix/default.nix

```
@@ -12,7 +12,7 @@ buildDunePackage rec {
     sha256 = "sha256-Xuw4pEPqAbQjSHrpMCNE7Th0mpbNMSxdEdwvH4hu2SM=";
   };
 
-  minimumOCamlVersion = "4.03";
+  minimalOCamlVersion = "4.03";
   useDune2 = true;
```

```suggestion
  duneVersion = "2";
```
This can also likely be 3